src/chrome/content/rules/LuaJIT.org.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	luajit.org has a wildcard DNS record, all other subdomains are mismatched.
+
+	Incomplete certificate chain:
+		- wiki
```

Comment:
I got a mismatch

Reply:
https://whatsmychaincert.com/?wiki.luajit.org

Reply:
Both is true. It's an incomplete, mismatching cert.

Reply:
✅